openeo/rest/connection.py

```diff
@@ -588,6 +617,30 @@ def describe_collection(self, name) -> dict:
         data = self.get('/collections/{}'.format(name)).json()
         return VisualDict("collection", data = data)
 
+    def collection_items(self, name, spatial_extent: Optional[Dict[str, float]] = None, temporal_extent: Optional[List[Union[str, datetime.datetime]]] = None) -> Paginate:
+        """
+        Loads items for a specific image collection.
+        May not be available for all collections.
+
+        :param name: String Id of the collection
+        :param spatial_extent: Limits the items to the given bounding box.
+        :param temporal_extent: Limits the items to the specified temporal interval.
+        The interval has to be specified as an array with exactly two elements (start, end).
+        Also supports open intervals by setting one of the boundaries to None, but never both.
+        :return: data_list: List A list of items
+        """
+        q = []
+        if spatial_extent is not None:
+            bbox = map(lambda c: str(c), spatial_extent)
+            q.append('bbox={}'.format(','.join(bbox)))
+        
+        if temporal_extent is not None:
+            datetimes = map(lambda t: '..' if t is None else rfc3339.normalize(t), temporal_extent)
+            q.append('datetime={}'.format('/'.join(datetimes)))
+        
+        url = '/collections/{cid}/items?{q}'.format(cid = name, q = '&'.join(q))
```

Comment:
I think we can avoid all this query parameter serialization, the `requests` library will take care of that for you. e.g

```python
params = {}
if spatial_extent:
    params["bbox"] = ",".join(str(c) for c in spatial_extent)
if temporal_extent:
    params["datetime"] = "/".join(".." if t is None else rfc3339.normalize(t) for t in temporal_extent)

con.get(url, params=params)
```

Reply:
Great, didn't know...